package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@easypost/api",
-      "version": "6.7.3",
+      "version": "6.8.1",
```

Comment:
Was this branch rebased correctly?

Reply:
Yes, why?

Reply:
Ooo, looks we forgot to bump the version in package-lock.json when releasing the `6.8.1`